Microsoft.Toolkit.Uwp.Services/Services/OneDrive/OneDriveStorageItem.cs

```diff
@@ -202,7 +203,8 @@ public OneDriveStorageItem(IOneDriveClient oneDriveProvider, IItemRequestBuilder
             // ParentReference null means is root
             if (oneDriveItem.ParentReference != null)
             {
-                _path = oneDriveItem.ParentReference.Path.Replace("/drive/root:", string.Empty);
+                if (oneDriveItem.ParentReference.Path != null)
```

Comment:
Can you merge the if statement with the parent one ex 
`if (oneDriveItem.ParentReference != null && oneDriveItem.ParentReference.Path != null)`
or you can use 
`if(oneDriveItem.ParentReference?.Path != null)`

Reply:
yes, your solution is better. I tested it and the OneDrive sample runs without problem now.

Reply:
Can you update the PR with your new changes ? :)